lib/filterx/func-str.c

```diff
@@ -65,6 +67,26 @@ typedef struct _FilterXExprAffix
   FilterXExprAffixProcessFunc process;
 } FilterXExprAffix;
 
+typedef struct _FilterXStrcasecmp
+{
+  FilterXFunction super;
+
+  union
+  {
+    GString *literal;
```

Comment:
I don't see much value in using a GString wrapper here, but that's only a nit, it adds an extra indirection at runtime.